docs/src/app/(public)/(content)/react/overview/quick-start/page.mdx

```diff
@@ -0,0 +1,42 @@
+# Quick Start
+
+<Subtitle>A quick guide to getting started with Base UI.</Subtitle>
+<Meta name="description" content="A quick guide to getting started with Base UI." />
+
+## Install the library
+
+Install Base UI using a package manager. All components are included in this single package.
+
+```jsx title="Terminal"
+npm i @base-ui-components/react
+```
+
+> All components are included in a single package. Base UI is tree-shakeable, so your app bundle will contain only the components that you actually use.
+
+## Set up portals
+
+Base UI uses portals for components that render popups, such as Dialog and Popover. To make portalled components always appear on top of the entire page, add the following style to your application layout root:
```

Comment:
Should we link to Dialog and Popover pages or do you think that's unnecessary?

Reply:
Popover is linked below. Distracting/unnecessary to have many links in a walkthrough.